test/Constraints/sr7098.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck %s
+// REQUIRES: objc_interop
+
+import Foundation
+
+class C : NSObject, NSWobbling {
+  func wobble() {}
+  func returnMyself() -> Self { return self }
+}
+
+func testDynamicOptionalRequirement(_ a: AnyObject) {
+  a.optionalRequirement?()
```

Comment:
Where is optionalRequirement defined?

Reply:
`test/Inputs/clang-importer-sdk/usr/include/Foundation.h` line 870.